docs/intro/overrides.rst

```diff
@@ -1,15 +1,20 @@
-.. _`intro-overrides`:
+.. _intro-overrides:
 
 Overrides
 =========
 
-Overrides contains mapping rules to associate which URLs a particular
-Page Object would be used. The URL matching rules is handled by another library
-called `url-matcher <https://url-matcher.readthedocs.io>`_.
+Overrides are rules represented by a list of :class:`~.ApplyRule` which
+associates which URL patterns a particular Page Object would be used. The URL
+matching rules is handled by another library called
+`url-matcher <https://url-matcher.readthedocs.io>`_.
 
 Using such rules establishes the core concept of Overrides wherein a developer
-could declare that a specific Page Object must be used *(instead of another)*
-for a given set of URL patterns.
+could declare that for a given set of URL patterns, a specific Page Object must
+be used instead of another Page Object.
+
+The :class:`~.ApplyRule` also supports pointing to the item returned by a specific
+Page Object if it both matches the URL pattern and the Item Class specified in the
+rule.
```

Comment:
> I would keep this introduction to explain in as few words as possible what an
override is

I'm not too sure about this one, as this section functions like an **abstract** which summarizes what to expect from a given paper (in this case, the doc section). 

> I think both of those can be introduced later as needed.

It will definitely be introduced later on. But we still need a high-level overview to provide the motivation of why this section of the doc is important.

> it may be a good idea to make the first reference to Page Object here
link to the Page Objects section in intro/from-ground-up.rst.

Good idea!

Reply:
I would avoid linking to stuff the used does not need to read at this point. But I have no strong opinion on this, so let’s move on.